network/wifi/wifi.c

```diff
@@ -244,6 +302,7 @@ int32_t wifi_init(struct wifi_desc **desc, struct wifi_init_param *param)
 	if (!ldesc)
 		return FAILURE;
 
+	ldesc->server.id = -1;
```

Comment:
Assigning -1 to uint32_t will work because of the compiler but it's probably not what you intended. Maybe use a `#define INVALID_SERVER_ID 0xffffffff` to make this code less cryptic.

Reply:
I used INVALID_ID